atest/testdata/parsing/test_case_settings.robot

```diff
@@ -180,10 +180,6 @@ Timeout
     [Timeout]    1d
     No Operation
 
-Timeout with message
-    [Timeout]    666    Message not supported since RF 3.2
-    No Operation
```

Comment:
Is there a test for using timeout with more than one value somewhere else? If not, this probably should be added back, renamed to something like "Invalid timeout" and validation on the `robot` side adjusted accordingly.

Reply:
Yes, there is a separate suite about timeouts, this test is completely redundant